src/main/resources/application.yml

```
@@ -0,0 +1,18 @@
+spring:
+  datasource:
+    url: jdbc:h2:~/test
+    username: sa
+    password:
+
+  h2:
+    console:
+      enabled: true
+
+  jpa:
+    generate-ddl: true
+    hibernate.ddl-auto: create-drop
+    show-sql: true
+    open-in-view: false
+    properties:
+        hibernate:
+        query.in_clause_parameter_padding: true
```

실수로 탭이 더 들어간 것 같습니다!
```suggestion
    properties:
      hibernate:
        query.in_clause_parameter_padding: true
```

---

매의눈..감사합니다🙇🏻‍♀️